h/presenters/annotation_json.py

```diff
@@ -60,10 +58,13 @@ def _get_read_permission(self):
             # It's not shared so only the owner can read it
             return self.annotation.userid
 
-        if security.Everyone in principals_allowed_by_permission(
-            AnnotationContext(self.annotation), Permission.Annotation.READ
+        # If the group is the public group, or an unauthorized person could
+        # read from it, it's world readable.
```

Comment:
```suggestion
        # If the annotation's group is the public group, or an unauthorized person could
        # read the annotation, then the annotation is world readable.
```

The "it's" reads like it refers to the group ("If the group is public ... it's world readable") but it actually refers to the annotation. Also added "then" to explicitly express the causal relationship.